lib/chat/utils/utils.dart

```diff
@@ -0,0 +1,36 @@
+String getExpiryTimeString(DateTime expiryTime) {
```

Comment:
Also rename this file as `time_utils.dart` as all the functions are time related.